frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -99,3 +99,7 @@ export enum REQUESTER_FILTER {
   USER = 'user',
   SYSTEM = 'system',
 }
+
+export const CLUSTER_VERSION_DEFAULT_UPSTREAM_SERVER_URL =
+  'https://api.openshift.com/api/upgrades_info/v1/graph';
```

Comment:
@spadgett for clarification, there are three scenarios (kudos to @wking):
1. spec.upstream is unset/empty (see https://github.com/openshift/installer/pull/4112)
1. spec.upstream is `https://example.com/some/custom/thing`
1. spec.upstream is explicitly set, but happens to match the CVO's internal default

This PR will treat 1 as the default case and 2/3 as custom case. The value in this constant is used for display purposes only and will never be sent to the backend.